test/relations.jl

```diff
@@ -703,7 +703,7 @@ end
 
         @test count(air_temperature.(ts_lower) .== Ref(_T_freeze)) ≥ 217
         @test count(air_temperature.(ts_upper) .== Ref(_T_freeze)) ≥ 217
-        @test count(air_temperature.(ts_mid) .== Ref(_T_freeze)) ≥ 1395
+        @test count(air_temperature.(ts_mid) .== Ref(_T_freeze)) ≥ 1392
```

Comment:
Strange to have these arbitrary numbers here. Fractions of total would be more justifiable.

Reply:
Yeah, we're using numbers just to be precise, this is just a heuristic, and it tells us that we're in fact testing the branch where we return `T_freeze`.